parents/integrationtest/pom.xml

```diff
@@ -185,45 +208,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>unpack-integrationtest-sharedresources</id>
```

Comment:
If you remove this, I think you can remove an (obsolete) comment in `documentation/pom.xml`:

```
<!-- Inherit execution unpack-shared-test-build-resources -->
```